packages/react-storage/src/components/StorageBrowser/createProvider.tsx

```diff
@@ -52,7 +51,6 @@ export default function createProvider<
           <ControlProvider actions={actions}>
             <LocationConfigProvider {...config}>
               <ActionProvider listLocationsAction={listLocationsAction}>
-                <Controller />
```

Comment:
I think we still need `Controller` for now since it's loading the data for location detail view? At the moment I think location detail view doesn't load any items since `<Controller />` was removed

Reply:
Oooooh, yeah good catch. Missed that in the rebase